hat/assets/js/apps/Iaso/components/forms/TextArea.tsx

```diff
@@ -122,6 +124,7 @@ export const TextArea: FunctionComponent<Props> = ({
                     setTextValue(e.target.value);
                 }}
                 value={textValue}
+                disabled={disabled}
```

Comment:
You need to apply specific style on the label if disabled and remove the withe background

Reply:
I made a separate ticket for this

Reply:
I actually fixed it and will close the additional ticket